samples/nrf5340/smp_svr/linker_arm_extxip.ld

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022 Carlo Caione <ccaione@baylibre.com>
+ * Copyrigt# Copyright (c) 2024 Nordic Semiconductor
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @file
+ * @brief External QSPI flash Linker command/script file
+ *
+ * Linker script for moving desired .text and and .data to the external
```

Comment:
We should re-think usage of external in documentation context; this script, for example, does not move anything to external memory space because the range is actually internal memory mapped to QSPI peripheral connected device.

Reply:
And this is MCUboot enabled application specific script as it reserves the MCUboot header. I know that this script will work with non-MCUboot app to, it will just waste the 512 bytes.